src/vm/contexts.rs

```diff
@@ -1043,6 +1043,18 @@ impl <'a> LocalContext <'a> {
             }
         }
     }
+
+    pub fn lookup_callable_contract(&self, name: &str) -> Option<&(QualifiedContractIdentifier, TraitIdentifier)> {
```

Comment:
This works for now -- but we can you add a comment here that eventually we'll want to be able to refer directly to the function's top-level context rather than recursing via parent? In theory, the cost of looking up a bound trait does not correspond to the stack depth (because they can only be defined at the top level of the function), but the current implementation does depend on stack depth.

Reply:
Addressed with https://github.com/blockstack/stacks-blockchain/pull/1644/commits/084ec6e3222e1ad5bd8c44775484acdfc79b6323